docs/src/pages/system/styled/styled.md

```diff
@@ -93,3 +93,56 @@ const styled = createStyled({ defaultTheme });
 
 export default styled;
 ```
+
+## Difference with the `sx` prop
+
+We want the `styled` function to behave similarly to the `styled` function of other technologies, like Emotion or Styled Components. 
+The `sx` prop however, is a new way of styling your application, focused on fast customization. `styled` is a function, while `sx` is a prop of Mui components.
+
+Therefore, you may observe a few differences:
+- `sx` provides more shortcuts than `styled`
+- the style definition slightly varies
+- patterns to use props are different
+
+You'll need to take those difference into account when refactoring your components.
+
+### Using props
+
+With `styled`:
+```js
+const MyStyledButton = styled("button")({
+  backgroundColor: props => props.myBackgroundColor
+})
```

Comment:
```suggestion
const MyStyledButton = styled("button")((props) => ({
  backgroundColor: props.myBackgroundColor
}))
```

So far, we've recommended this approach, it is much cleaner in my opinion, especially when there are longer styles. We use the same when building the MUI components

Reply:
Completely agree, this is also the only syntax accepted by `tss-react` for the migration of `makeStyles`. However `makeStyles` documentation [here](https://mui.com/styles/api/#returns-3) is showing the "per-field" approach. So it's not obvious for people coming from v4 whether both will work/which one is recommended